frappe/website/doctype/website_settings/website_settings.py

```diff
@@ -129,7 +129,7 @@ def get_items(parentfield):
 		where parent='Website Settings' and parentfield= %s
 		order by idx asc""", parentfield, as_dict=1)
 
-	top_items = [d for d in all_top_items if not d['parent_label']]
+	top_items = [d for d in all_top_items]
```

Comment:
Earlier, there was a difference, as `top_items` were only the first level parents.
Now, there is no difference, this will allow us to build a multi-level tree of Top Bar Items, which we can use to build a multilevel dropdown.

Reply:
Also, list comprehension is used because we want to make a copy of the list and deepcopy doesn't seem to work

Reply:
If you're only looking to create a copy of the list, you can use list slicing as:

```python
top_items = all_top_items[:]
```